src/fees/hooks.ts

```diff
@@ -32,3 +42,43 @@ export function usePaidFees(fees: Fee[]) {
     totalFee,
   }
 }
+
+// Returns the estimated fee for a SEND or INVITE transaction
+// also optionally fetches new fee estimations if the current ones are missing or out of date
+export function useEstimatedFee(
+  tokenAddress: string,
+  feeType: FeeType.SEND | FeeType.INVITE,
+  shouldRefresh: boolean = true
+) {
```

Comment:
I feel the hook name + description is misleading now.
It only returns the estimated fee to send a transaction when the token passed as input matches the token that would be used to pay for fees.
Otherwise it's 0, but it doesn't mean the fee is actually 0 in the final transaction.

The existing use case was to be able to send full amounts without failing (because you wouldn't have enough balance to pay for the fee if you sent your full balance in a token that would also be used to pay for the fee).

I don't have a good naming proposition though.
Maybe reuse the existing one + description?